packages/rum-core/src/domain/rumEventsCollection/view/trackViewEventCounts.ts

```diff
@@ -0,0 +1,32 @@
+import { ONE_MINUTE } from '@datadog/browser-core'
+import type { LifeCycle } from '../../lifeCycle'
+import { trackEventCounts } from '../../trackEventCounts'
+
+// Arbitrary delay for stopping event counting after the view ends. Ideally, we would not stop and
+// keep counting events until the end of the session. But this might have a small performance impact
+// if there are many many views: we would need to go through each event to see if the related view
+// matches. So let's have a fairly short delay to avoid impacting performances too much.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what’s best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** It might be good to explain a little more the issue

```suggestion
// Some events are not being counted as they transcend views. To reduce the occurrence; 
// an arbitrary delay is added for stopping event counting after the view ends. 
// 
// Ideally, we would not stop and keep counting events until the end of the session.
// But this might have a small performance impact if there are many many views: 
// we would need to go through each event to see if the related view matches.
// So let's have a fairly short delay to avoid impacting performances too much.
```